docs/source/topics/frontier-settings.rst

```diff
@@ -202,6 +202,15 @@ Default: ``crawlfrontier.utils.fingerprint.sha1``
 
 The function used to calculate the ``domain`` fingerprint.
 
+.. setting:: OVERUSED_SLOT_FACTOR
+
+OVERUSED_SLOT_FACTOR
+------------------------
+
+Default: ``1.0``
+
+This is an amount of overall active connections to concurrent connections allowed in slot(per-domain/ip) after which slot
+ is considered overused.
```

Comment:
It's a tricky topic. It can be greater and and it can be lower, it all depends on what URL's spider sent there at some moment in time. Concurrency can be affected by CONCURRENT_REQUESTS_PER_IP/DOMAIN. Number of active requests in the slot are influenced by delay (which is adaptive with AUTOTHROTTLE), remote web resource request processing time, or even how overloaded with parsing the whole crawler process. The default meaning is "consider slot overused when count of active connections is more than concurrent allowed for that slot". If you have concurrent=10, and active=11 slot is overused.